application/common/mail/password-pwned.html.php

```diff
@@ -18,7 +18,7 @@
 <p>
     This email is to inform you that the password you use with your <?= $idpDisplayName ?> identity (IdP) account has
     been discovered in a database of leaked passwords. This does not specifically mean your <?= $idpDisplayName ?>
```

Comment:
Is `$idpDisplayName` more like "Acme" or "Acme IdP"?

If the former, we may want to change `your <?= $idpDisplayName ?> account` to `your <?= $idpDisplayName ?> identity account`, to help protect against people misinterpreting this to be referring to their corporate email account's password.

Reply:
yeah, and to be consistent with the use right above it. I'll identity in that second instance.